src/posts/paper/13-q-and-a.ts

```diff
@@ -212,16 +212,37 @@ While the vaccines are nowhere near making you invulnerable to COVID, remember t
 
-Additionally, we have seen [data](https://pubs.rsna.org/doi/full/10.1148/ryct.2020200110) suggesting that asymptomatic COVID-19 could result in lung damage in half of cases.
+We have not done a thorough analysis of this effect in other vaccines.
 
-As a result, getting a vaccination yourself confers a reduction in incoming microCOVIDs (as outlined [above](#1-how-much-less-likely-is-it-for-vaccinated-individuals-to-catch-covid-reduction-in-microcovids)),
-but we do not recommend increasing your microCOVID budget on top of that given current data.
+We have not seen data that describes the effect of vaccines on long COVID in mild/asymptomatic cases. The 1% annual risk budget is based on risks of long term effects of
+COVID (as opposed to hospitalization or death). Reports of prevalence in long COVID in unvaccinated people varies widely:
+* A [study by ZOE](https://www.medrxiv.org/content/10.1101/2020.10.19.20214494v1.full.pdf) found that "one in twenty people (5%) had COVID-19 symptoms for more than eight weeks, and around 2% of people can experience symptoms for 12 weeks or more". The data for this was self-reported and the researchers noted that many participants stopped reporting data before the conclusion of the study.
+* Another [study](https://pubs.rsna.org/doi/full/10.1148/ryct.2020200110) early in the pandemic found lung abnormalities in half of asymptomatic COVID-19 cases on the Diamond Princess cruise ship. These lung abnormalities are similar to pneumonia; it is unclear from that study what percent led to long term health changes.
+* Researchers found [heart abnormalities in 78 of 100 recovered patients](https://jamanetwork.com/journals/jamacardiology/fullarticle/2768916), sampling an average of 71 days after initial diagnosis. There may be self-selection bias among participants in this cohort.
 
-If you do decide you want to increase your microCOVID budget post vaccination,
-* Discuss this with any regular close contacts you have, such as housemates, partners, or co-workers — your increased risk of COVID, asymptomatic or not, could affect them more than it affects you.
-* Consider the potential impact on your community at large — taking on more risk of being an asymptomatic spreader could result in you infecting many other people.
+As a result, getting a vaccination yourself confers a reduction in incoming microCOVIDs based on how the vaccine affects your chances of infecting others (as outlined [above](#1-how-much-less-likely-is-it-for-vaccinated-individuals-to-catch-covid-reduction-in-microcovids)).
+You may additionally increase your budget based on how the vaccine reduces your own chances of negative outcomes, but this is a personal choice dependent on your own view of how averse to the various possible negative outcomes you are.
+
+Some ways you may want to change your budget in response to getting vaccinated people around you vaccinated:
```

Comment:
Just a small clarity comment, apologies if this isn't helpful: "getting vaccinated people around you vaccinated" on line 230 seems redundant/unclear? Did we mean to say:

`"...in response to people around you getting vaccinated"?`

Reply:
Thanks for catching that typo!